spec/std/array_spec.cr

```diff
@@ -1319,136 +1340,200 @@ describe "Array" do
   end
 
   describe "sort" do
-    it "sort without block" do
-      a = [3, 4, 1, 2, 5, 6]
-      b = a.sort
-      b.should eq([1, 2, 3, 4, 5, 6])
-      a.should_not eq(b)
-    end
+    [true, false].each do |stable|
```

Comment:
Use tuples over arrays for less amount of heap allocations:

```suggestion
    {true, false}.each do |stable|
```

ditto elsewhere

Reply:
Doesn't matter at all in specs